.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["@babel/preset-flow"]
```

Comment:
What does this do, exactly?

Reply:
In the meantime, I have looked at what this preset does and don't think it fits. Do you think another preset would fit? If so, why?

Reply:
It's the recommended preset when using flow:

- https://babeljs.io/docs/en/babel-preset-flow
- https://flow.org/en/docs/tools/babel/ (css is currently broken)

That being said, we can use a different plugin to parse the flow annotations with Babel: https://babel.dev/docs/en/babel-plugin-syntax-flow (last commit)

As for "Why?", it looks like more recent versions of babel and/or the eslint parser require some config to parse some of the code in the test suite like here: https://github.com/willdurand/eslint-plugin-no-unsanitized/blob/06cce63a4f6c4d82131e9abf01abe09cd18001fb/tests/rules/property.js#L522

Reply:
I don't follow. We don't use flow or babel when authoring the plugin. Babel is only used as a library when parsing typescript source code. Why do we need this?

Reply:
Isn't babel parser imported in the test file I linked above? The line pointed by this same link shows some code with flow annotations. This config is used for the test suite only.

Reply:
Maybe we are talking past each other. We don't use type annotations ourselves. The code you mentioned is only within a JS string. Can you please remove the file?

Reply:
FTR the discussion finished in Slack. It wasn't super clear from the patch that we had test cases covering flow annotations and that the babel parser was used in this context.